google-cloud-bom/pom.xml

```diff
@@ -1052,11 +1052,21 @@
         <artifactId>opencensus-api</artifactId>
         <version>${opencensus.version}</version>
       </dependency>
+      <dependency>
```

Comment:
We should never depend on the impl artifact. The end user should do this.

Reply:
Should we consider GCS team as the end user? I mean, they might want to collect the data by default. I figure it is better to involve GCS team member in the discussion here. @jabubake @frankyn

Reply:
The end user is the user that uses this client to talk to GCS.

Reply:
Yes. And I agree that it provides more flexibility. I'll remove it for now and leave that decision for GCS team.

Reply:
I don't understand the package structure of opencensus very well. What's the consequence of removing the impl artifact?

Reply:
We have opencensus-api which provides basic API and no-op implementation for trace and stats, while opencensus-impl, if present in classpath, will replace the op with real implementation.

So the idea here is that we should always depend on no-op api by default, and let end user to optionally enable the tracing/stats by adding opencensus-impl.

Reply:
I see, this makes sense. Thank you for confirming.